Formula/sfk.rb

```diff
@@ -12,12 +12,10 @@ class Sfk < Formula
   end
 
   def install
-    # Using the standard ./configure && make install method does not work with sfk as of this version
-    # As per the build instructions for macOS, this is all you need to do to build sfk
-    system ENV.cxx, "-DMAC_OS_X", "sfk.cpp", "sfkext.cpp", "-o", "sfk"
-
-    # The sfk binary is all you need. There are no man pages or share files
-    bin.install "sfk"
+    system "./configure", "--prefix=#{prefix}"
+    chmod 0755, "install-sh"
```

Comment:
Cool. Can you please submit this as a bug to the upstream developers of this project and add a link to the upstream bug submission and explanation of why the chmod is needed in a comment in the formula file. Thanks!